components/02-elements/button/_button.scss

```diff
@@ -1,36 +1,37 @@
 $button-height                    : 48px !default;
 $button-width                     : 48px !default;
-$button-padding                   : 0 35px !default;
+$button-padding                   : 0 $padding-base !default;
 $button-margin                    : 0 0 5px 0 !default;
 $button-border                    : none !default;
+$button-border-radius             : 100px !default;
 $button-hover-border              : none !default;
 $button-background                : $color-primary !default;
-$button-hover-background          : darken($color-primary, 20%) !default;
+$button-hover-background          : $gray-dark !default;
 $button-outline                   : none !default;
-$button-font-family               : $font-family-sans-serif !default;
+$button-font-family               : $font-family-base-bold !default;
 $button-font-size                 : $font-size-base !default;
 $button-font-color                : $white !default;
-$button-hover-font-color          : $white !default;
+$button-hover-font-color          : $color-primary !default;
 
 // Secondary
-$button-secondary-border          : 3px solid $gray-light !default;
+$button-secondary-border          : 1px solid $color-primary !default;
 $button-secondary-hover-border    : $color-primary !default;
 $button-secondary-background      : $white !default;
-$button-secondary-hover-background: $white !default;
-$button-secondary-font-color      : $gray-light !default;
-$button-secondary-hover-font-color: $color-primary !default;
+$button-secondary-hover-background: $color-primary !default;
+$button-secondary-font-color      : $color-primary !default;
+$button-secondary-hover-font-color: $gray-dark !default;
 
 // Icon
 $button-icon-padding              : 0 !default;
 $button-icon-size                 : 20px !default;
-$button-icon-content-margin       : 0 15px 0 0 !default;
+$button-icon-content-margin       : 0 $margin-base 0 0 !default;
 $button-icon-border-width         : 3px !default;
-$button-icon-border               : $button-icon-border-width solid $gray-light !default;
-$button-icon-hover-border         : $color-primary !default;
-$button-icon-fill                 : $gray-light !default;
+$button-icon-border               : $button-icon-border-width solid $color-primary !default;
+$button-icon-hover-border         : $gray-dark !default;
+$button-icon-fill                 : $white !default;
 $button-icon-hover-fill           : $color-primary !default;
-$button-icon-background           : $white !default;
-$button-icon-hover-background     : $white !default;
+$button-icon-background           : $color-primary !default;
+$button-icon-hover-background     : $gray-dark !default;
 $button-icon-color                : $gray-light !default;
 $button-icon-hover-color          : $color-primary !default;
```

Comment:
please use BEM notification for variables

Reply:
Done